frontend/packages/console-shared/src/hooks/formik-validation-fix.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { useFormikContext } from 'formik';
+import { useDeepCompareMemoize } from './deep-compare-memoize';
+
+export const useFormikValidationFix = (value) => {
```

Comment:
```suggestion
export const useFormikValidationFix = (value: any) => {
```

Not a huge fan of this idea, but I would prefer an explicit `any` instead of an implicit `any`. We, far too often, forget types. So I'd prefer to see an explicit any rather than nothing.

Reply:
Updated with this - 
```
export const useFormikValidationFix = (value: Partial<FormikValues>) => {
```